### DISCUSSION TITLE
运行utils.utils当中的extract_ubfc_dataset()函数报错

### DISCUSSION THREAD

**Discussion by BugMaker2002** [Dylan-H-Wang/SLF-RPM#8] [Q&A] [answered]
Hello! When I try to run the extract_ubfc_dataset() function in utils.utils, I get the following error:
![image](https://github.com/Dylan-H-Wang/SLF-RPM/assets/110211332/dec59eee-491c-4aa2-a5fe-b5058396a405)
How can I solve this problem?

**Comment by Dylan-H-Wang** [comment, marked as answer]
Check this https://github.com/OpenTalker/SadTalker/issues/611#issuecomment-1705801360

**Comment by BugMaker2002** [comment]
thank you very much
